### ISSUE TITLE
Portal Link to ESS-DIVE Data

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/nmdc-server#371]
Provide hyperlink to project record in ESS-DIVE

Priority - High
Urgency - Medium

**Comment by jbeezley** [comment]
We likely need this information added to the schema and the workflow pipelines. Once that is done, it's a quick task to add it to the UI.

**Comment by kfagnan** [comment]
This will require a bit more work/collaboration with the ESS-DIVE team to figure out what makes sense for cross-linking. The first test case will be WHONDERS (Stegen project). 
@emileyfadrosh @pvangay - could this be part of the Stegen FICUS outreach? Either way, I'm going to remove from the May sprint...

**Comment by pvangay** [comment]
Happened to have a call with Joan today. It looks like ESS DIVE data links are project based -- so we should be able to link to DOIs that point to ESS DIVE datasets (we already do this for Brodie's landing page). For links by sample, Joan suggested linking to SESAR. But agree - by sample links are not a priority for May.

**Comment by pvangay** [comment]
FYI Joan put together a [spreadsheet](https://docs.google.com/spreadsheets/d/1HyWhwsOWD6np8MrEl9gzD3jpLmyd_oTNtbmTw28O0tM/edit#gid=652587412) of SESAR (per sample) and ESS DIVE links (per dataset) for the Brodie dataset - putting this here so folks know how things are linked.

**Comment by aclum** [comment]
This was completed last year for GSP. WHONDERS datasets in ESS-DIVE are linked to GROW (nmdc study id nmdc:sty-11-5tgfr349). Will spin out a different ticket to connect SESAR (per sample) and ESS DIVE links (per dataset) for the Brodie dataset.
